src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute used to binds a parameter to an DocumentDB type.
+    /// </summary>
+    /// <remarks>
+    /// The method parameter type can be one of the following:
+    /// <list type="bullet">
+    /// <item><description><see cref="ICollector{T}"/></description></item>
+    /// <item><description><see cref="IAsyncCollector{T}"/></description></item>
```

Comment:
You bring up a good point. This worked for EasyTables b/c JObject was inherently supported. Let me start writing the script code and make sure it works E2E in node before I move forward.

---
In reply to: [56370824](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/47#discussion_r56370824) [](ancestors = 56370824)